api/handlers/jobs.ts

```diff
@@ -270,5 +272,5 @@ function getPreviewUrl(payload: Payload | undefined, env: string): string | unde
   const { repoOwner, branchName, project } = payload;
   const githubUsernameNoHyphens = repoOwner.split('-').join('').toLowerCase();
   const possibleStagingSuffix = env === 'stg' ? 'stg' : '';
-  return `https://preview-mongodb${githubUsernameNoHyphens}${possibleStagingSuffix}.gatsbyjs.io/${project}/${branchName}/index`;
```

Comment:
This has been fixed on the source plugin side, so making the change here to reflect this